setup.cfg

```diff
@@ -0,0 +1,9 @@
+# This section allows us to run
+# python setup.py test
+[aliases]
+test=pytest
+
+# This section defines setup for tests
+[tool:pytest]
+addopts = --verbose
+python_files = tests/*/*.py
```

Comment:
check wildcards to ensure that all test will be run

Reply:
ensure that it is not reliant on there being exactly 2 levels of folders

Reply:
Pytest uses glob-style file pattern: https://docs.pytest.org/en/latest/reference.html#confval-python_files
It looks like we needs double stars: https://facelessuser.github.io/wcmatch/glob/